src/local/butler/reproduce.py

```diff
@@ -16,41 +16,114 @@
 from __future__ import print_function
 from builtins import object
 
+import httplib2
+import json
 import os
 import shutil
 import tempfile
+import urllib
+import webbrowser
 
+from src.python.base import utils
 from src.python.bot.tasks import commands
 from src.python.fuzzing import testcase_manager
 from src.python.system import environment
 from src.python.system import shell
 
+# TODO(mbarbella): This should work cross platform.
+AUTHORIZATION_CACHE_FILE = os.path.join(
+    os.path.expanduser('~'), '.config', 'clusterfuzz', 'authorization-cache')
 
-class _SimplifiedTestcase(object):
+# TODO(mbarbella): Don't use the old clusterfuzz-tools client id.
+OAUTH_URL = 'https://accounts.google.com/o/oauth2/v2/auth?%s' % (
+    urllib.urlencode({
+        'scope':
+            'email profile',
+        'client_id': ('981641712411-sj50drhontt4m3gjc3hordjmp'
+                      'c7bn50f.apps.googleusercontent.com'),
+        'response_type':
+            'code',
+        'redirect_uri':
+            'urn:ietf:wg:oauth:2.0:oob'
+    }))
+
+# TODO(mbarbella): This value should come from the configuration.
+TESTCASE_URL = 'https://clusterfuzz.com/reproduce-tool/testcase-info'
```

Comment:
put something in data_handler ? It defined several _URL globals and uses get_domain()

Reply:
I'd like to address this in a later change. We do have functions like get_domain but this wouldn't work with only the test config (which is all someone would have in a fresh checkout). We're going to need to come up with a solution here, but I don't think there's a quick fix for this one.